src/features/Search/moviesAPI.js

```diff
@@ -0,0 +1,53 @@
+import { key, url } from "../../core/apiCodes";
+import { MovieDetails } from "../movies/MovieDetails";
+import { selectPopularMoviesList } from "../MoviesList/popularMoviesSlice";
```

Comment:
It's cool that we have a folder with all api codes, but are they needed in this code?